### PR TITLE
dialog when opening remote datasets

### PR DESCRIPTION
**Pull request by kcarra**

### CODE REVIEW

File: src/components/generic-elements/call-to-action-button/call-to-action-button.test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    Object.defineProperty(window, "location", {
+      value: {
+        href: url
+      },
+      writable: true
+    });
+
+    let subject = shallow(
+      <CallToActionButton url={"abc"} format="csv" filename="dataset.csv" />
+    );
```
Comment (cameronmarsh) [inline src/components/generic-elements/call-to-action-button/call-to-action-button.test.js:24]: this is a small thing, but I would like to see us use the pattern of the other tests where we have a function with overrides for default params. IMO, it makes it easier to see what the test is actually testing

Comment (kcarra) [reply src/components/generic-elements/call-to-action-button/call-to-action-button.test.js:24]: You got it Jobin!

File: src/components/generic-elements/call-to-action-button/call-to-action-button.test.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    subject.find("a").simulate("click");
```
Comment (cameronmarsh) [inline src/components/generic-elements/call-to-action-button/call-to-action-button.test.js:26]: I kind of forget, but can you use className attributes with `find`? I think that would also make it clearer what this is doing

File: src/components/generic-elements/call-to-action-button/call-to-action-button.test.js (hunk 1 above)
Comment (cameronmarsh) [reply src/components/generic-elements/call-to-action-button/call-to-action-button.test.js:24]: 😎

**Pull request merged**